### PR TITLE
Added global config

### PR DESCRIPTION
**Pull request by stasya72008**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+class Config:
+    def __init__(self, section):
+        self._parser = configparser.RawConfigParser()
+        self._config_file = conf_file
+        self._section = ''
+        try:
+            self._parser.read(self._config_file)
+        except configparser.Error, exc:
+            raise configparser.Error("Error reading config file %r: %s" %
```
Comment (denyskulykov) [inline config.py]: Please, don't use this 'zooland' for formating string like **%**

File: config.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                                     (self._config_file, str(exc)))
+
+        if not self._parser.has_section(section):
+            print 'Available sections:'
```
Comment (denyskulykov) [inline config.py]: Please, don't use prent, this cause for logger =)

File: config.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import configparser
+from os import path
+
+conf_file = path.join(path.dirname(__file__), 'config.ini')
```
Comment (denyskulykov) [inline config.py]: Maybe better, move it to **init()**

File: config.ini [hunk 4]
```diff
@@ -0,0 +1,11 @@
+[global]
+
+[rest]
+host = 185.160.62.217
+port = 5000
+debug = False
+
+[calendar]
+host = 185.160.62.217
+port = 5000
```
Comment (denyskulykov) [inline config.ini]: Need to set different ports for each application +)

File: config.py (hunk 2 above)
Comment (stasya72008) [reply config.py]: Re-write this sentence correctly, not in Indian style, please =)

File: config.py (hunk 3 above)
Comment (stasya72008) [reply config.py]: Comma in not needed here

Comment (stasya72008) [reply config.py]: Besides, why is it better?

File: config.py (hunk 2 above)
Comment (denyskulykov) [reply config.py]: Please, don't use _print_
Using of logger is better here
:-P

File: config.py [hunk 5]
```diff
@@ -0,0 +1,74 @@
+import configparser
+from os import path
+
+
+class Config:
+    def __init__(self, section):
+        conf_file = path.join(path.dirname(__file__), 'config.ini')
```
Comment (denyskulykov) [inline config.py]: Please, drop this line)
and re-white 9 line - 
self._config_file = path.join(path.dirname(__file__), 'config.ini')

File: web_app/rest_client/tasks.py [hunk 6]
```diff
@@ -1,11 +1,12 @@
 import requests
 import json
 
+import config
 import data.data_templates as template
 
-# ToDo(stasya) Move base_url to config after
-# https://github.com/stasya72008/highway-to-hell/projects/2#card-11155636
-base_url = 'http://localhost:5000/'
+conf = config.CalendarConfig()
```
Comment (denyskulykov) [inline web_app/rest_client/tasks.py]: Here we should use conf for Rest, 
rest_client calls rest, no wep_app

**Pull request merged**